card/card.html

```diff
@@ -0,0 +1,270 @@
+
+            </div>
+            
+        </div>
+
+
+        <h2 class = "heading">Card with dismiss</h2>
+        <p>
+            Lorem ipsum dolor sit amet, consectetur adipiscing elit. Ut at orci at
+            justo imperdiet mollis nec vitae erat.
+            facilisis.
+          </p>
+
+        <div class="basic-card basic-card-vertical border-radius-5px card-dismiss">
+
+            <div class = "img-section">
+                <img class ="card-img card-flex-item border-radius-5px" src="/images/card-img.jpeg">
+                <button class="btn-wishlist border-radius-5px"><i class="bi bi-heart-fill"></i></button>
+                <span class="dismiss-on-card"><i class="bi bi-x-square-fill"></i></span>
+            </div>
+            
+            <div class="card-body">
+
+                <p>Trendy Elegant Women Tops & Tunics</p>
+
+                <div class = "price-details">
+                    <span class = "new-price">&#x20B9 158</span>
+                    <span class = "old-price">&#x20B9 225</span>
+                    <span class = "offer">30% off</spam>
+                </div>
+               
+                 <div class = "rating">
+                    <span>3.9</span>
+                    <span><i class="bi bi-star-fill"></i></span>
+                 </div>
+                
+                <button class="btn-addtocart border-radius-5px"><i class="bi bi-cart"></i>Add To Cart</button>
+                
+
+            </div>
+            
+        </div>
+        
+
+        <h2 class = "heading">Card with text overlay</h2>
+        <p>
+            Lorem ipsum dolor sit amet, consectetur adipiscing elit. Ut at orci at
+            justo imperdiet mollis nec vitae erat.
+            facilisis.
+          </p>
+
+        <div class="basic-card basic-card-vertical border-radius-5px">
+            <div class = "text-overlay-container">
+
+                <div class = "img-section">
+                    <img class ="card-img card-flex-item border-radius-5px" src="/images/card-img.jpeg">
+                    <button class="btn-wishlist border-radius-5px"><i class="bi bi-heart-fill"></i></button>
+                    
+                </div>
+                
+                <div class="card-body">
+    
+                    <p>Trendy Elegant Women Tops & Tunics</p>
+    
+                    <div class = "price-details">
+                        <span class = "new-price">&#x20B9 158</span>
+                        <span class = "old-price">&#x20B9 225</span>
+                        <span class = "offer">30% off</spam>
+                    </div>
+                   
+                     <div class = "rating">
+                        <span>3.9</span>
+                        <span><i class="bi bi-star-fill"></i></span>
+                     </div>
+                    
+                    <button class="btn-addtocart border-radius-5px"><i class="bi bi-cart"></i>Add To Cart</button>
+                    
+    
```

Comment:
here too

Reply:
> code is good, just remove spacing in html file where not needed. And give spacing after css property so that it's easier to read the css values. good to merge with suggested changes.

I had made changes as suggested. Thanks for the review.